docs/userguide/data.rst

```diff
@@ -3,7 +3,9 @@
 Data management
 ===============
 
-Parsl enables implementation of dataflow patterns in which data passed among apps manage the flow of execution. Dataflow programming models are popular as they can cleanly express, via implicit concurrency, opportunities for parallel execution.
+Parsl supports staging of file data between execution locations and at-rest
+locations. This supports two pieces of functionality: execution location
```

Comment:
this sounds a little like the staging is one-way, and not the way I expected - maybe say more explicitly that staging is from at-rest to execution for inputs and back for outputs